libraries/plugins/peerplays_sidechain/peerplays_sidechain_plugin.cpp

```diff
@@ -120,7 +123,7 @@ void peerplays_sidechain_plugin_impl::plugin_initialize(const boost::program_opt
       config_ready_son = config_ready_son && !_sons.empty();
 
 #ifndef SUPPORT_MULTIPLE_SONS
-      FC_ASSERT( _sons.size() == 1, "Multiple SONs not supported" );
+      //FC_ASSERT( _sons.size() == 1, "Multiple SONs not supported" );
```

Comment:
Watch out for this. Use:
 cmake -DBOOST_ROOT="$BOOST_ROOT" -DCMAKE_BUILD_TYPE=Debug -DSUPPORT_MULTIPLE_SONS=1

Reply:
Taken care off.